src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -331,12 +332,23 @@ project-with-fastly-complex:
         ports:
             - 80
         fastly:
+            shield: true
```

Comment:
Might actually be better as `default-shield` now?

Reply:
That may be unclear if `backends` isn't used, so if the intention is to have a value here and for `backends.*` to override it, keeping the same name is consistent with other configurations